index.html

```diff
@@ -8286,11 +8286,12 @@ <h3>Ticks</h3>
  content observable effects that must be consistent across
  implementations. To accommodate this, the specification requires
  that <a>remote ends</a> <dfn>perform implementation-specific action
- dispatch steps</dfn> on a <a>browsing context</a> <var>context</var>,
- and a <var>list of events</var> and their properties. These steps
- must be equivalent to performing the given input device manipulations
- on <var>context</var>, such that trusted events corresponding to the
- entries in <var>list of events</var>are dispatched.
+ dispatch steps</dfn> on a <var>context</var>, and a <var>list of events</var>
```

Comment:
I think we should keep <a>browsing context</a> indicating the context var type

Reply:
Yeah, I missed it.